src/main/java/bio/terra/service/dataset/flight/ingest/IngestParseJsonFileStep.java

```diff
@@ -88,7 +88,7 @@ public StepResult doStep(FlightContext flightContext)
                         .map(
                             columnName -> {
                               JsonNode fileRefNode = node.get(columnName);
-                              if (fileRefNode.isObject()) {
+                              if (!fileRefNode.isNull() && fileRefNode.isObject()) {
```

Comment:
Is this the line that was NPEing?  wouldn't it still?  Like is `fileRefNode` itself null?